api/test/multiaddrs.test.js

```diff
@@ -0,0 +1,42 @@
+import test from 'ava'
+import { findUsableMultiaddrs } from '../basic/helper/multiaddr.js'
+
+const fixture = [
+  '/ip4/127.0.0.1/tcp/4001/p2p/12D3KooWADjHf2kyANQodg9z5sSdX4bGEMbWg7ojwu6SCyDAMtzN',
```

Comment:
Could we add a dns4 address just to make sure everything is working as expected for non hasBogonIpAddress?